### PR TITLE
Add docu for undocumented functions

### LINKED ISSUE #181: Document duration template funcs
See https://github.com/gohugoio/hugo/pull/3870

### LINKED ISSUE #739: Document strings.Contains and other missing strings methods

### LINKED ISSUE #1898: Document crypto.FNV32a

### PR DESCRIPTION
**Pull request by deining**
This PR adds doucmentation for string manipulation functions, duration functions and the `crypto.FNV32a` function.
This PR closes #1898.
This PR closes #181.
This PR closes #739.

### CODE REVIEW

File: content/en/functions/strings.ToLower.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (jmooring) [inline content/en/functions/strings.ToLower.md:1]: The already documented "lower" function is an alias for strings.ToLower. You can remove this page.

You are welcome to add a second signature to the existing function. Here's an example:
https://gohugo.io/functions/replacere

I keep this link handy to find aliases: https://raw.githubusercontent.com/gohugoio/hugo/master/docs/data/docs.json

File: content/en/functions/strings.ToUpper.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (jmooring) [inline content/en/functions/strings.ToUpper.md:1]: The already documented "upper" function is an alias for strings.ToUpper. You can remove this page.

You are welcome to add a second signature to the existing function.

**Review by jmooring — CHANGES_REQUESTED** [review]
Thank you for this!

**Comment by jmooring** [comment]
One more thing... your commit is not addressing issues, it is closing them!  

Closes #181
Closes #739
Closes #1898

File: content/en/functions/crypto.FNV32a.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+title: crypto.FNV32a
+description: Calculate the FNV (Fowler–Noll–Vo) 32 bit hash of a given string.
```
[the remark this answers is not in the document]
Comment (deining) [reply content/en/functions/crypto.FNV32a.md:3]: Changed as requested.

File: content/en/functions/duration.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+aliases: []
+---
+
+`time.Duration` converts a given number into a [`time.Duration`](https://pkg.go.dev/time#Duration) structure so you can access its fields. E.g. 
+you can perform [time operations](https://pkg.go.dev/time#Duration) on the returned `time.Duration` value:
+
+    {{ printf "There are %.0f seconds in one day." (duration "hour" 24).Seconds }}
+    <!-- Output: There are 86400 seconds in one day. -->
+
+Also, you can use the `duration` function with [Hugo pipes](https://gohugo.io/hugo-pipes/introduction/):
```
[the remark this answers is not in the document]
Comment (deining) [reply content/en/functions/duration.md:20]: Replaced the sentence as requested.

File: content/en/functions/crypto.FNV32a.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    parent: "functions"
+keywords: [crypto hash FNV32 Fowler-Noll-Vo]
+signature: ["crypto.FNV32a STRING"]
+hugoversion:
+aliases: []
+---
+
+This function calculates the 32 bit [FNV1a hash](https://en.wikipedia.org/wiki/Fowler%E2%80%93Noll%E2%80%93Vo_hash_function#FNV-1a_hash) of a given string according to the [specification](https://datatracker.ietf.org/doc/html/draft-eastlake-fnv-12):
+
+    {{ crypto.FNV32a "Hello world" }} → "1498229191"
```
[the remark this answers is not in the document]
Comment (deining) [reply content/en/functions/crypto.FNV32a.md:16]: This is corrected now.

File: content/en/functions/duration.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+    {{  mul 7.75 60 | duration "minute"}} → 7h45m0s
+    {{  mul 120 60 | mul 1000 | duration "millisecond"}} → 2h0m0s
+
+You have to specify a time unit for the number given to the function. Valid time units are:
+* nanoseconds,
```
[the remark this answers is not in the document]
Comment (deining) [reply content/en/functions/duration.md:26]: Replaced the example with a table, as requested.

File: content/en/functions/strings.Contains.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
+---
+title: strings.Contains
+description: Check if a string contains a substring.
```
[the remark this answers is not in the document]
Comment (deining) [reply content/en/functions/strings.Contains.md:3]: Changed as requested.

File: content/en/functions/duration.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
+---
+title: duration
+description: Converts the given number to a `time.Duration` structure, using the given time unit.
+categories: [functions]
+menu:
+  docs:
+    parent: "functions"
+keywords: [time duration]
+signature: ["duration TIME_UNIT"]
+hugoversion:
```
[the remark this answers is not in the document]
Comment (deining) [reply content/en/functions/duration.md:10]: Line removed as requested.

File: content/en/functions/strings.Contains.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+categories: [functions]
+menu:
+  docs:
+    parent: "functions"
+keywords: [string strings substring contains]
+signature: ["strings.Contains STRING SUBSTRING"]
+hugoversion: ""
```
[the remark this answers is not in the document]
Comment (deining) [reply content/en/functions/strings.Contains.md:10]: Line removed as requested.

File: content/en/functions/duration.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+`time.Duration` converts a given number into a [`time.Duration`](https://pkg.go.dev/time#Duration) structure so you can access its fields. E.g. 
+you can perform [time operations](https://pkg.go.dev/time#Duration) on the returned `time.Duration` value:
+
+    {{ printf "There are %.0f seconds in one day." (duration "hour" 24).Seconds }}
+    <!-- Output: There are 86400 seconds in one day. -->
+
+Also, you can use the `duration` function with [Hugo pipes](https://gohugo.io/hugo-pipes/introduction/):
+
+    {{  mul 7.75 60 | duration "minute"}} → 7h45m0s
+    {{  mul 120 60 | mul 1000 | duration "millisecond"}} → 2h0m0s
```
[the remark this answers is not in the document]
Comment (deining) [reply content/en/functions/duration.md:23]: Spacing corrcted as requested.

File: content/en/functions/strings.ContainsAny.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
+---
+title: strings.ContainsAny
+description: Check if a string contains any character from a given string.
```
[the remark this answers is not in the document]
Comment (deining) [reply content/en/functions/strings.ContainsAny.md:3]: Changed as requested.

File: content/en/functions/strings.ContainsAny.md [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+categories: [functions]
+menu:
+  docs:
+    parent: "functions"
+keywords: [string strings substring contains any]
+signature: ["strings.ContainsAny STRING CHARACTERS"]
+hugoversion: ""
```
[the remark this answers is not in the document]
Comment (deining) [reply content/en/functions/strings.ContainsAny.md:10]: Removed line as requested.

File: content/en/functions/strings.Contains.md [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+relatedfuncs: [strings.ContainsAny]
+---
+
+    {{ strings.Contains "Hugo" "go" }} → true
+
+The check is case sensitive: 
+
+    {{ strings.Contains "Hugo" "Go" }} → false
+
+Numbers can be handled as well:
```
[the remark this answers is not in the document]
Comment (deining) [reply content/en/functions/strings.Contains.md:21]: Section removed as requested.

File: content/en/functions/strings.ContainsAny.md [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+relatedfuncs: [strings.Contains]
+---
+
+    {{ strings.ContainsAny "Hugo" "gm" }} → true
+
+The check is case sensitive: 
+
+    {{ strings.ContainsAny "Hugo" "Gm" }} → false
+
+Numbers can be handled as well:
```
[the remark this answers is not in the document]
Comment (deining) [reply content/en/functions/strings.ContainsAny.md:21]: Section removed as requested.

File: content/en/functions/strings.ToLower.md (hunk 1 above)
Comment (deining) [reply content/en/functions/strings.ToLower.md]: Page removed and second signature added to the existing function.

File: content/en/functions/strings.ToUpper.md (hunk 2 above)
Comment (deining) [reply content/en/functions/strings.ToUpper.md]: Page removed and second signature added to the existing function.

**Comment by deining** [comment]
> One more thing... your commit is not addressing issues, it is closing them!
> 
> Closes #181 Closes #739 Closes #1898

I edited the description of the PR, these three issues are now linked with this PR and will be closed once this PR was merged.

File: content/en/functions/crypto.FNV32a.md [hunk 15]
```diff
@@ -0,0 +1,16 @@
+---
+title: crypto.FNV32a
+description: Calculate the FNV (Fowler–Noll–Vo) 32 bit hash of a given string.
+categories: [functions]
+menu:
+  docs:
+    parent: "functions"
+keywords: [crypto hash FNV32 Fowler-Noll-Vo]
+signature: ["crypto.FNV32a STRING"]
+hugoversion:
```
[the remark this answers is not in the document]
Comment (deining) [reply content/en/functions/crypto.FNV32a.md]: Line was remooved as requested.

**Pull request merged**